### ISSUE TITLE
Boiler temperature stopped working

### ISSUE THREAD

**Issue by Gyosa3** [msvisser/remeha_home#16]
Hi,

when I switched to latest version the temperature was added and in device `Remeha DHW` and it worked well.

Since a week or 2 I noticed that it stopped working and the entity `sensor.remeha_dhw_water_temperature` is "unavailable".

As it worked for a while, I suspect that Remeha changed something in the API +/- recently, or it's something on my side, but I tried to reinstall the integration code to no avail.

Ciao,
G.

**Comment by msvisser** [comment]
For my personal system the water temperature sensor is still working, so I think it is unlikely the API was changed. Is the water temperature sensor the only thing that is no longer working?

**Comment by Gyosa3** [comment]
Thanks. Something was wrong with my setup even after I redownloaded the package and restarted everything several times. I saw this because the mode of my boiler was "frost_protection" not "Frost protection", which meant that the new translation file was not taken into account. So it went to the newer version then back to the older version for some reason.

I fixed the problem by force deleting the whole folder, integation and custom component and re-install everything. Maybe that was a remainder of the different test I did some time ago. Now it works well again. Thanks!
